tokio-macros/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.2.1 (December 18, 2019)
+
+### Fixes
+- maintain fn visibility when transforming (#1954).
```

Comment:
Maybe better wordin would be `Inherit fn visibility when wrapping function`

Reply:
tweaked.